### ISSUE TITLE
automation support for long read assembly

### ISSUE THREAD

**Issue by aclum** [microbiomedata/nmdc_automation#82]
There have been changes to support long read workflows. A new wdl, choose_assembly.wdl, with a boolean value for if the short or long read workflow should run is in development. Once this is merged into main with a tagged release the automation code should be updated to call this wdl with the new input requirements. 

Depends on https://github.com/microbiomedata/issues/issues/454

@kaijli
